ui/pages/home/index.scss

```diff
@@ -152,4 +152,8 @@
     background: none;
     margin-left: 20px;
   }
+
+  &__error-message {
+    left: 10px;
```

Comment:
Adhere to [8px grid](https://consensys.slack.com/archives/G1L7H42BT/p1637778098226300)
```suggestion
    left: 8px;
```

Reply:
the other sides end up being 10 from here:
https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/app/multiple-notifications/index.scss#L59-L60

![image](https://user-images.githubusercontent.com/364566/143505174-efc506e9-e775-489a-8ed0-4471ba93d802.png)

Reply:
Ah ok. I would push for changing them all to be `8px` but if it's outside the scope of this PR keep it 10px to be consistent.